snmp/datadog_checks/snmp/data/default_profiles/juniper-pulse-secure.yaml

```diff
@@ -0,0 +1,113 @@
+extends:
+  - _generic-host-resources-base.yaml
+  - _generic-if.yaml
+  - _generic-ucd.yaml
+  - _juniper.yaml
+sysobjectid: 1.3.6.1.4.1.12532.256.*
+metrics:
+  - MIB: JUNIPER-IVE-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.12532.1.0
+      name: juniper.ive.logFullPercent
+  - MIB: JUNIPER-IVE-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.12532.2.0
+      name: juniper.ive.signedInWebUsers
+  - MIB: JUNIPER-IVE-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.12532.3.0
+      name: juniper.ive.signedInMailUsers
+  - MIB: JUNIPER-IVE-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.12532.10.0
+      name: cpu.usage
+  - MIB: JUNIPER-IVE-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.12532.11.0
+      name: memory.usage
+  - MIB: JUNIPER-IVE-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.12532.12.0
+      name: juniper.ive.iveConcurrentUsers
+  - MIB: JUNIPER-IVE-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.12532.13.0
+      name: juniper.ive.clusterConcurrentUsers
+  - MIB: JUNIPER-IVE-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.12532.14.0
+      name: juniper.ive.iveTotalHits
+    metric_type: monotonic_count
+  - MIB: JUNIPER-IVE-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.12532.15.0
+      name: juniper.ive.iveFileHits
+    metric_type: monotonic_count
+  - MIB: JUNIPER-IVE-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.12532.16.0
+      name: juniper.ive.iveWebHits
+    metric_type: monotonic_count
+  - MIB: JUNIPER-IVE-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.12532.17.0
+      name: juniper.ive.iveAppletHits
+    metric_type: monotonic_count
+  - MIB: JUNIPER-IVE-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.12532.18.0
+      name: juniper.ive.ivetermHits
```

Comment:
According to the MIB, no:
```
ivetermHits OBJECT-TYPE
    SYNTAX      Counter64   
    MAX-ACCESS  read-only
    STATUS  current
    DESCRIPTION  
    "The Total number of terminal hits to the IVE since last reboot"
    ::= { juniper-ive 18 }